.github/ISSUE_TEMPLATE/icon_request.yml

```
@@ -0,0 +1,85 @@
+name: Icon request
+description: Request a new icon for Simple Icons
+title: "Request: "
+labels: [new icon]
+
+body:
+  - type: markdown
+    attributes:
+      value: >-
+        We won't add non-brand icons or anything related to illegal services. If in
+        doubt, open an issue and we'll have a look. Before opening a new issue, make
+        sure it isn't covered by an existing issue. Please search for [issues with the
+        `new icon`
+        label](https://github.com/simple-icons/simple-icons/issues?q=is%3Aissue+label%3A%22new+icon%22+is%3Aopen)
+        (including closed issues) before you continue. If you find one for the brand
+        you're requesting then leave a comment on it or add a reaction to the original
+        post. Also take a look at our [contributing
+        guidelines](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md)
+        for more details.
+
+  - type: input
+    attributes:
+      label: Brand Name
+      placeholder: "Example: Simple Icons"
+    validations:
+      required: true
+
+  - type: input
+    attributes:
+      label: Website
+      description: >-
+        For non-web brands you can add a relevant link or simply put "None" if you don't
+        think there's a website.
+      placeholder: "Example: https://simpleicons.org"
+    validations:
+      required: true
+
+  - type: input
+    attributes:
+      label: Popularity Metric
+      description: >-
+        Provide either an [Alexa rank](https://www.alexa.com/siteinfo) or the number
+        of GitHub stars. If these are not available or insufficient you can still open
```

I think we can simplify the language used here:

```suggestion
        of GitHub stars. If these are not available or too low you can still open
```